source/_integrations/zwave_me.markdown

```diff
@@ -93,7 +93,13 @@ Z-Wave.Me Z-Way requires Z-Wave.Me hardware:
 
 ## Installing Z-Way
 
-Z-Wave.Me Z-Way runs on various platforms: Raspberry Pi OS, Linux, Windows. Installation instructions are available here: https://z-wave.me/z-way/download-z-way/
+Z-Way can be installed as standalone as well as in a supervised Home Assistant, HAOS and on Home Assitsant Yellow controller.
+
+Check [Z-Wave.Me Add-on repository](https://github.com/Z-Wave-Me/ha-z-wave-me-addon) or use the installation button
+
+[![Open your Home Assistant instance and show the add add-on repository dialog with a specific repository URL pre-filled.](https://my.home-assistant.io/badges/supervisor_add_addon_repository.svg)](https://my.home-assistant.io/redirect/supervisor_add_addon_repository/?repository_url=https%3A%2F%2Fgithub.com%2Fz-wave-me%2Fha-z-wave-me-addon)
```

Comment:
This is a third-party add-on. We don't support third-party add-ons and therefore do not list them in our official documentation.

The reason is they might indicate a recommendation or endorsement, but the reality is: We don't support them.

Reply:
> We need to explain somehow how to install Z-Way on the same box (or on Yellow).

That is not in the scope of Home Assistant. We also don't explain how to install your Hue hub, for example.

> Z-Wave JS is an official add-on? Do you maintain it or is it the community?

That is an official add-on provided and maintained by the Home Assistant project.

Reply:
Ok, will know that. Anyway, this is strange as this is the only way to install staff on Yellow.